tests/integration/targets/grafana_dashboard/tasks/dashboard-from-file.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: copy dashboard file
```

Comment:
Any reason for this copy ?

Reply:
yes, the dashboard.json file should be present on the remote server, unless with use a `delegate_to: localhost` for the import task

Reply:
It makes more sense to use the `delegate_to: localhost` since we are using the API.